flowexperimental/flownewtonmethod.hpp

```diff
@@ -0,0 +1,266 @@
+// -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
```

Comment:
this file should be called flowexpnewtonmethod (and class named accordingly). as is you'd think it was used by flow.